setup.cfg

```diff
@@ -91,19 +94,16 @@ console_scripts =
     briefcase = briefcase.__main__:main
 briefcase.platforms =
     android = briefcase.platforms.android
-    # django = briefcase.platforms.django
     iOS = briefcase.platforms.iOS
     linux = briefcase.platforms.linux
     macOS = briefcase.platforms.macOS
     # tvOS = briefcase.platforms.tvOS
     # watchOS = briefcase.platforms.watchOS
     # wearos = briefcase.platforms.wearos
+    web = briefcase.platforms.web
     windows = briefcase.platforms.windows
 briefcase.formats.android =
     gradle = briefcase.platforms.android.gradle
-# briefcase.formats.django =
-#     project = briefcase.platforms.django.project
-#     app = briefcase.platforms.django.app
```

Comment:
Throwing this out there while we're here...

Do you want to remove `homebrew` as well for now?

Reply:
Oddly, the homebrew case is slightly different. The Django backend never really existed; there was a code module, but it wasn't loaded. The homebrew *does* exit, and is actually being used by tests. It probably shouldn't be, as it's a bit of a red herring... but it's a little difficult to extract.

Reply:
Ok - you've successfully nerdsniped me :-)